src/Publisher.ts

```diff
@@ -147,9 +165,9 @@ export default class Publisher {
 					path,
 				},
 			);
-			// @ts-expect-error
-			if (response.status === 200 && response.data.type === "file") {
-				// @ts-expect-error
+			// @ts-expect-error TODO: abstract octokit response
+			if (response.status === 200 && response?.data.type === "file") {
+				// @ts-expect-error TODO: abstract octokit response
```

Comment:
just leaving a note that all octokit-things should be abstracted to a class / functions (so it can be switched out / response dealt with outside of application logic) 

leaving fixing types to that day